server/src-lib/Hasura/GraphQL/Utils.hs

```diff
@@ -38,6 +39,12 @@ getBaseTy = \case
   where
     getBaseTyL = getBaseTy . G.unListType
 
+unwrapTy :: G.GType -> G.GType
```

Comment:
This isn't a very descriptive name, but that fits with the codebase so I move on.